### ISSUE TITLE
couple of clarifications

### ISSUE THREAD

**Issue by sophia-wright-blue** [qfgaohao/pytorch-ssd#12]
Thank you for releasing this repo. could you clear up a couple of questions:

1). what is the url to download the VGG based SSD? (for e.g., mobilenet based SSD is at https://storage.googleapis.com/models-hao/mobilenet-v1-ssd-mp-0_675.pth)

2). could you provide some details of how you trained the model - 'gun_model_2.21.pth'? Can I use 'gun_model_2.21.pth' with the VGG based SSD?

3). Have you worked with YOLO based models? I was wondering if you could provide any insight on the practical difference in accuracy between YOLO and SSD models?

Thanks again for releasing this repo. Look forward to your reply.

**Comment by qfgaohao** [comment]
Hi @sophia-wright-blue For your first question, I replaced the Scaled L2 Norm layer with Batchnorm in VGG16-ssd recently in order to make it ONNX and Caffe2 compatible. I will try to train and upload a model with reasonable accuracy next week.

For the second question, definitely you can VGG16. 

In terms of YOlO, I've tested V1 and V2. They are fast. But they tend to have much more false positives compared with SSD models. I haven't tried V3 yet. I know YOLO has good benchmark results on Pascal VOC data.  So feel free to try it on your dataset.

Cheers!

**Comment by sophia-wright-blue** [comment]
thanks for your reply and for providing a detailed README file for your repo :)

its an interesting point about false positives with YOLO. 

i have a broad general related question, hope you don't mind me asking here:

do you think we can train an SSD model to distinguish between a person's different poses in an image? right now, SSD detects a bounding box around a person. what if we fed in images of a person sitting and person standing during training? could we then get SSD to distinguish between a person sitting and a person standing?

thanks again for your reply,

**Comment by qfgaohao** [comment]
Hi @sophia-wright-blue , it's possible to use SSD for pose detection if you have enough examples as training data. At the same time, Dense Pose or R-Mask might be a better option if your focus is mainly on accuracy rather than speed. 

Good luck with your project!

**Comment by sophia-wright-blue** [comment]
thanks for replying @qfgaohao , i'm looking to do object detection and pose detection in one model, will look into this some more.
